spec.md

```diff
@@ -320,32 +329,32 @@ For success responses, the following fields are valid.
 }
 </pre>
 
-### Polling Interval and Duration    
+### Polling Interval and Duration
 
-The frequency and maximum duration of polling may vary by platform client. If a platform has a max polling duration and this limit is reached, the platform will cease polling and the operation state will be considered `failed`.
+The frequency and maximum duration of polling MAY vary by platform client. If a platform has a max polling duration and this limit is reached, the platform will cease polling and the operation state will be considered `failed`.
 
 ## Provisioning
 
-When the broker receives a provision request from the platform, it should take whatever action is necessary to create a new resource. What provisioning represents varies by service and plan, although there are several common use cases. For a MySQL service, provisioning could result in an empty dedicated database server running on its own VM or an empty schema on a shared database server. For non-data services, provisioning could just mean an account on an multi-tenant SaaS application.
+When the broker receives a provision request from the platform, it MUST take whatever action is necessary to create a new resource. What provisioning represents varies by service and plan, although there are several common use cases. For a MySQL service, provisioning could result in an empty dedicated database server running on its own VM or an empty schema on a shared database server. For non-data services, provisioning could just mean an account on an multi-tenant SaaS application.
 
 ### Request ###
 
 ##### Route #####
 `PUT /v2/service_instances/:instance_id`
 
-The `:instance_id` of a service instance is provided by the platform. This ID will be used for future requests (bind and deprovision), so the broker must use it to correlate the resource it creates.
+The `:instance_id` of a service instance is provided by the platform. This ID will be used for future requests (bind and deprovision), so the broker will use it to correlate the resource it creates.
```

Comment:
I think I swapped this from a "must" to a "will" because how the correlation is done is an impl detail. Clearly the intent is for people to use the ID, that's why it's there, but if someone uses some other mechanism behind the scenes does that mean they're violating the spec?  For example, what if each client is only allowed one service - then the ID becomes meaningless and they may not use it. If someone looks at an impl's code and notices that, but everything works as expected, are they non-compliant?   The other reason I thought MUST may not be right is that I wasn't sure whether this is testable w/o looking at the code.  But I'd like to hear what others think.